src/error.rs

```diff
@@ -165,6 +165,8 @@ pub enum RoutingError {
     Cbor(CborError),
     /// invalid response
     Response(ResponseError),
+    /// crust error
+    Crust(::crust::error::Error),
```

Comment:
I think you should be able to use `::crust::Error` here (http://maidsafe.net/crust/master/crust/index.html#reexports).